src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -130,19 +162,84 @@ const Backdrop = styled('div')<Openable>`
   }};
 `
 
+const renderUpsellCard = (
```

Comment:
Why not write a normal react component `UppsellCard` => `<UppsellCard {...props}>`?

Reply:
Right. So the idea was to try to keep things easy to remove as this is being defined as an experimental thing. But to honest I guess things got out of hand and there's no way to easily remove this anymore. So I guess I can just do what you've suggested.

Reply:
haha sure sure, I just mean this is a functional component already -- we just don't use a variable name that suggests it is 😅